x-pack/plugins/ml/kibana.json

```diff
@@ -17,7 +17,8 @@
     "uiActions",
     "kibanaLegacy",
     "indexPatternManagement",
-    "discover"
+    "discover",
+    "maps"
```

Comment:
This should probably go into `optionalPlugins` and then you'll need to check that `maps` is not `undefined`.

Reply:
Made it optional here https://github.com/elastic/kibana/pull/88880/commits/fb0f337bd5aa9dc645ac6b690809808bb20082c8